content/200-orm/500-reference/500-preview-features/050-client-preview-features.mdx

```diff
@@ -66,7 +66,7 @@ In the list below, you can find a history of Prisma Client and Prisma schema fea
 | [`referentialIntegrity`](/orm/prisma-schema/data-model/relations/relation-mode)                                            | [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |  [4.7.0](https://github.com/prisma/prisma/releases/tag/4.7.0)  |
-| [`interactiveTransactions`](/orm/prisma-client/queries/transactions#interactive-transactions)                              | <ul><li>[2.29.0](https://github.com/prisma/prisma/releases/tag/2.29.0)</li><li>with Data Proxy [4.6.0](https://github.com/prisma/prisma/releases/tag/4.6.0)</li></ul>                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |  [4.7.0](https://github.com/prisma/prisma/releases/tag/4.7.0)  |
```

Comment:
This also applies to Accelerate. Do we not support that far back?

Reply:
With Accelerate we support interactive transactions from v`5.1.1`

Reply:
Ah thanks Ankur! I've added it in the **Released into General Availability** column.